content/swift/concepts/dictionaries/terms/removeAll/removeAll.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '.removeAll()'
+Description: 'Removes all key-value pairs in a dictionary.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Dictionaries'
```

Comment:
Let's change this to be a tag already in the [tags document](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) which is "Dictionary".
```suggestion
  - 'Dictionary'
```

Reply:
Fixed

Reply:
Fixed